.pre-commit-config.yaml

```
@@ -0,0 +1,50 @@
+ci:
+  autoupdate_schedule: monthly
```

```suggestion
  autoupdate_schedule: quarterly
```

I prefer this but not very strongly. It's nice to not get pinged for updates that make no difference.